examples/medical-center/Ballerina.toml

```diff
@@ -3,15 +3,21 @@ org = "wso2"
 name = "medical_center"
 version = "0.1.0"
 
-[[dependency]]
-org = "ballerina"
-name = "persist"
-version = "0.1.0"
-repository = "local"
-
 [persist.medical_center.storage.mysql]
 host = "localhost"
 port = 3305
 user = "root"
 password = "Test123#"
 database = "medical_center_db"
```

Comment:
```suggestion
[persist]
datastore = "mysql"
module = "medical_center"
```

Reply:
Fixed in [3ac4df4](https://github.com/ballerina-platform/module-ballerina-persist/pull/155/commits/3ac4df4447e4c445dda487eb932865f44b7f670c)